### DISCUSSION TITLE
ChannelPointsMiner video tutorial

### DISCUSSION THREAD

**Discussion by Ra91995** [RakambdaOrg/ChannelPointsMiner#300] [Q&A]
Hi, can the author make a video tutorial ?
I don't understand the text

**Comment by LeoAlt8** [comment]
I support! I would like a video with the installation from the very beginning.

**Comment by Rakambda** [comment]
I do understand that for most this project isn't really accessible. However I'm not a fan of making a video that will become obsolete some time from now because things changed.
[Plus right now with recent Twitch changes, it's quite random if your account will work or not]

The readme is very bulky, I do agree. I'm thinking on how to make it more readable but didn't really found a nice solution for now.

My stand on the project is that I have it more geared towards more experience users/with some knowledge. If you want a simple solution, Tkd's project is probably a better choice. My version basically adds multi account support and is more customizable in some areas. That customization adds the complexity.

---

Now as a mini guide on things, the easiest solution is probably to use docker (lots of resources online to learn it) and everything you need to start it up is [given in the readme](https://github.com/RakSrinaNa/ChannelPointsMiner#docker).
As for the global settings, same thing, [examples are given in the readme](https://github.com/RakSrinaNa/ChannelPointsMiner/tree/main/src/examples/config) and more streamer settings [can be found here](https://github.com/RakSrinaNa/ChannelPointsMiner/tree/main/miner/src/test/resources/factory).

I'd rather help people in Discord/Discussions when they have a "specific" question about things they tried, rather than taking people by the hand on how to do things. You can see it as a rather not nice answer, but you also have to put in consideration that I already spent quite a lot of time to do the readme, and even if it's not the most understandable, you can still get things out of it.

---

Though if someone from the community wants to make a video on it and explains the process, I'm fine with promoting it on the readme/
